### PR TITLE
Fungal towers (etc.) butcher yelds

### LINKED ISSUE #19937: Fungal tower corpse yields absolutely nothing upon butchering.
A bit odd for a 1.2 ton creature, if I can say so.

### PR DESCRIPTION
**Pull request by nexusmrsep**
Closes #19937
_[No yield when butchering fungal towers]_

Fungal towers, and other big fungal "structures" with persistent corpses now yield veggetable matter, like some other fungaloids.

There were two ways to resolve this issue:
**[1]** Some of the bigger fungaloids are made of "stone". I could have changed their material to "veggie" so the yieald-on-butcher parsers would correctly extract "body parts" assigned to vegetable matter. I have not done it this way however, becouse I believe that 'stone' material was seton purpose, as 'veggie' is handled differently in few places, for example when dealing with fire, etc.

**[2]** **_(applied here)_** I have added harvest yield table called "fungaloid_mass" (compare #23029) and added this harvest group to all fungaloid monsters that have both: "stone" body (ie. are unseen for standard parser functions for flesh/body parts) and normal deaths (ie. butcherable corpses).

Yield quantity can be discussed further from this point, but i tried to make it a bit larger then other huge creature's corpses, asi believe fungal towers, spires, and blossoms are collossal.

### CODE REVIEW

**Comment by nexusmrsep** [comment]
Sorry for the error with commits and reverst. I messed up here.

**Comment by BorkBorkGoesTheCode** [comment]
Try making a backup branch for this PR then git reset --hard 9dde6ed5be70f2806b0d95319264b1b2812e7d7b

File: data/json/monsters.json [hunk 1]
```diff
@@ -2263,6 +2263,7 @@
     "armor_cut": 10,
     "special_attacks": [ [ "FUNGUS_SPROUT", 10 ] ],
     "death_function": [ "FUNGUS", "NORMAL" ],
+    "harvest":"fungaloid_mass",
```
Comment (ZhilkinSerg) [inline data/json/monsters.json]: Better add whitespace after semicolon.

Comment (nexusmrsep) [reply data/json/monsters.json]: Typo, correcting.

File: data/json/monsters.json [hunk 2]
```diff
@@ -2315,6 +2317,7 @@
     "special_attacks": [ [ "FUNGUS_FORTIFY", 10 ] ],
     "death_drops": "marloss_yellow_drops",
     "death_function": [ "FUNGUS", "NORMAL" ],
+    "harvest":"fungaloid_mass",
```
Comment (DracoGriffin) [inline data/json/monsters.json]: ... and here. I think that's all.

File: data/json/monsters.json [hunk 3]
```diff
@@ -2287,6 +2288,7 @@
     "luminance": 200,
     "special_attacks": [ [ "FUNGUS_BIG_BLOSSOM", 10 ] ],
     "death_function": [ "FUNGUS", "NORMAL" ],
+    "harvest":"fungaloid_mass",
```
[the remark this answers is not in the document]
Comment (nexusmrsep) [reply data/json/monsters.json]: done...

File: data/json/monsters.json (hunk 2 above)
Comment (nexusmrsep) [reply data/json/monsters.json]: ... ough...missed so many. Done.

**Pull request merged**